data/World/Gerudo Training Ground MQ.json

```diff
@@ -30,6 +30,7 @@
         "locations": {
             "Gerudo Training Ground MQ Dinolfos Chest": "True",
             # Shoot the Gerudo symbol above the door to the lava room.
+            "Gerudo Training Ground MQ Dinolfos Arrow Wonderitem": "can_use(Bow)",
```

Comment:
According to the logic you can get to this region with a slingshot, does the wonderitem require bow or does slingshot work here as well? (The description says "certain damage type", so it probably is bow only, I just want to make sure)

Reply:
the wonderitem requires bow.